app/scripts/directives/sf-aloha-format-style.js

```diff
@@ -17,6 +17,19 @@ angular.module('authoringEnvironmentApp').directive(
                 );
                 element.toggleClass('disabled', isDisabled);
                 element.bind('click', function () {
+
+                    // this block sets the Aloha.activeEditable
+                    // to the editable of the current Aloha.Selection
+                    // needed due to a bug with the formatBlock function
```

Comment:
was that bug already fixed in newer releases ? upgrading aloha comes into play ?